crates/scroll/hardforks/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+//! Scroll-Reth hard forks.
+
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/paradigmxyz/reth/main/assets/reth-docs.png",
+    html_favicon_url = "https://avatars0.githubusercontent.com/u/97369466?s=256",
+    issue_tracker_base_url = "https://github.com/paradigmxyz/reth/issues/"
+)]
+#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
```

Comment:
Can we replace this with documentation relating to the scroll hardfork upgrades. If you download the `.md` file [here](https://github.com/scroll-tech/scroll-documentation/blob/develop/src/content/docs/en/technology/overview/scroll-upgrades.mdx), include it in the repo and reference it using `#![doc = include_str!("../docs/hardforks.md")]`.